compiler/rustc_interface/src/passes.rs

```diff
@@ -416,7 +416,7 @@ pub fn configure_and_expand(
     }
 
     if sess.opts.unstable_opts.hir_stats {
-        hir_stats::print_ast_stats(&krate, "POST EXPANSION AST STATS");
+        hir_stats::print_ast_stats(&krate, "POST EXPANSION AST STATS", "ast-stats-2");
```

Comment:
nit: `ast-stats-2` could probably be `ast-pre-stats` or something a little more descriptive? Likewise with `ast-stats-1`.

Reply:
I'll leave it as is. Anything short of `ast-stats-{pre,post}-expansion` is going to be non-obvious to a first-time user, but that's a bit on the long side. And the heading ensures that a first-time user will be able to quickly work out the meaning of the `1` and `2`.